packages/web-components/src/components/va-radio-option/va-radio-option.tsx

```diff
@@ -42,25 +45,24 @@ export class VaRadioOption {
   })
   radioOptionSelected: EventEmitter;
 
-  private handleChange(): void {
+  private handleChange = (): void => {
     this.radioOptionSelected.emit();
-  }
+  };
 
   render() {
-    const ariaDescribedby = (this.checked && this.ariaDescribedby) || null;
+    const { checked, handleChange, label, name, value } = this;
+
     return (
-      <Host
-        aria-checked={this.checked ? `${this.checked}` : 'false'}
-        aria-describedby={ariaDescribedby}
```

Comment:
See our [Slack discussion](https://dsva.slack.com/archives/G01BJ3ESXL4/p1650291953580479) about this attribute.